bugbug/models/defect_enhancement_task.py

```diff
@@ -27,7 +27,7 @@ def get_labels(self):
             "{} tasks".format(sum(1 for label in classes.values() if label == "task"))
         )
 
-        return classes
+        return classes, sorted(list(set(classes.values())), reverse=True)
```

Comment:
Same here, we have a defined set of labels (`task`, `enhancement` and `defect`).